src/humanize/number.pyi

```diff
@@ -0,0 +1,12 @@
+from typing import Any, List, SupportsInt, SupportsFloat, Tuple, Union, Optional
+
+def ordinal(value: Union[str, SupportsInt]) -> str: ...
```

Comment:
`ordinal()` is an example of a function that if the passed value can't be converted into an `int`, it is just returned, meaning the return type isn't always true. There are a number of functions like this, probably stemming from Django's filters that should fail gracefully rather than loudly.

Either the function's execution is changed so the returned value has `str()` called on it, meaning it's guaranteed to be a string (although not one that may be formatted nicely), the function throws an exception instead (which is a big change in terms of expected behaviour), or the return type is changed to `Any`. (Advice for python typing strongly recommends the use of Any over a Union returntype).